app/controllers/posts_controller.rb

```diff
@@ -1,15 +1,35 @@
 class PostsController < ApplicationController
-  before_action :set_user
-
   def index
-    @posts = Post.all
+    @user = User.find(params[:user_id])
+    @posts = @user.posts.order(created_at: :asc)
+  end
+
+  def new
+    @post = Post.new
+  end
```

Comment:
- When I try to add a new post with this URL it shows me that can't create post! so kindly solve this error here ⬇✨🤩 and check why is prohibting from creating new posts 
![image](https://user-images.githubusercontent.com/58553711/222934596-f60f9f11-0055-4406-9a46-c153ce804285.png)

Reply:
I checked it is working, please first create some users, then check other features.
![post](https://user-images.githubusercontent.com/60324033/222935653-f72df20c-e095-4c27-9016-5b437ad0137c.PNG)